matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt

```diff
@@ -413,5 +415,29 @@ internal class RealmSessionStoreMigration @Inject constructor(
                 chunkEntities.deleteAllFromRealm()
             }
         }
+
+        realm.schema.get("RoomSummaryEntity")
```

Comment:
Please add a `migrateTo21()` fun to avoid breaking database for people using latest develop version

Reply:
Done + rebase

Reply:
Thanks